anomalib/core/callbacks/normalization.py

```diff
@@ -0,0 +1,111 @@
+"""Anomaly Score Normalization Callback."""
+import copy
+from typing import Any, Dict, Optional
+
+import pytorch_lightning as pl
+import torch
+from pytorch_lightning import Callback, Trainer
+from pytorch_lightning.utilities.types import STEP_OUTPUT
+from torch.distributions import LogNormal, Normal
+
+
+class OutputNormalizationCallback(Callback):
```

Comment:
How about the name of your PR? :) Like `AnomalyScoreNormalizationCallback` or `ScoreNormalizationCallback`. It doesn't matter if it's verbose. What's more important is its readability